api/lib/source/cosmosV0-source.js

```diff
@@ -238,19 +238,28 @@ class CosmosV0API extends RESTDataSource {
       '/staking/pool'
     )
     if (!Array.isArray(response)) return []
-    const proposals = response.map((proposal) => {
-      return this.reducers.proposalReducer(
-        this.network.id,
-        proposal,
-        {},
-        totalBondedTokens
-      )
-    })
+    const proposals = await Promise.all(
+      response.map(async (proposal) => {
+        const [tally, proposer] = await Promise.all([
+          this.query(`/gov/proposals/${proposal.id}/tally`),
+          this.query(`gov/proposals/${proposal.id}/proposer`).catch(() => {
+            return { proposer: `unknown` }
```

Comment:
```suggestion
            return { }
```

Reply:
not sure about this but it seems odd to return `unknown`

Reply:
yeah, this was code from before actually. I also found it odd